testcli.go

```diff
@@ -127,9 +128,18 @@ func (test Test) do(ctx context.Context, binpath string, timeout time.Duration)
 		return err
 	}
 	for _, line := range bytes.Split(buf, []byte{'\n'}) {
+		// give time to usql to react to sent commands so statements are close to their results in output
+		// this helps link failures to the statement that caused them
+		time.Sleep(100 * time.Millisecond)
 		if err := exp.Send(string(line) + "\n"); err != nil {
 			return err
 		}
+		command := strings.TrimSpace(string(line))
+		if command == "\\q" || command == "\\quit" {
```

Comment:
String literal can help readability

```suggestion
		if command ==`\q` || command == `\quit` {
```